### ISSUE TITLE
Camera Problems

### ISSUE THREAD

**Issue by TJWhitehead** [BCLab-UNM/Swarmathon-ROS#62]
Two of our rovers require a reinstall of the ROS/Swarmathon camera drivers every time the NUC is booted up.  

Once the drivers are installed, and the NUC is left in a powered state, everything operates. 

ROS has been removed and reinstalled in an attempt to maintain the operation of the camera. 

Is anyone else having this issue?

Is there a way to fix this?

**Comment by jhecker** [comment]
This is certainly a new problem to me -- can you take a picture of the camera driver reinstall process that occurs when you reboot the NUC?

The simplest fix would be to perform a full reinstallation of Ubuntu, but let's check a few things before we try that.

**Comment by TJWhitehead** [comment]
It may have to be later today before I can get images. 

However, we(my professor, TA and project partners) have done a fresh install of Ubuntu on one of the rovers recently after the SSD failed according to the ROS indigo and swarmathon information on the GitHub site.

The commands that were used to reinstall the camera drivers are:

sudo apt-get install ros-indigo-usb-cam -- reinstall the drivers fresh
catkin_make clean in the rover workspace
then catkin_make.

I may be missing a few steps that we have used.

Yesterday removal of ROS and all directories were removed and reinstalled and still showing giving the same issue.

Maybe we are missing something and considering performing a file comparison form the operational rover with the rovers that don't have camera operation.

Thank you for the quick response and definitely look forward to more suggestions.
